packages/backend-for-frontend/src/services/catalogService.ts

```diff
@@ -1053,6 +1053,24 @@ export function catalogServiceBuilder(
 
       const previousDescriptor = retrieveLatestDescriptor(eService.descriptors);
 
+      if (eService.templateRef) {
+        const { id } = await catalogProcessClient.createInstanceDescriptor(
+          {
+            audience: [],
+            dailyCallsPerConsumer: previousDescriptor.dailyCallsPerConsumer,
+            dailyCallsTotal: previousDescriptor.dailyCallsTotal,
+            agreementApprovalPolicy: previousDescriptor.agreementApprovalPolicy,
+          },
```

Comment:
If these params are always set to these values, you could remove the params and retrieve the values from the previous descriptor directly in the catalog-process.
Also, the fact that we get these from the previous descriptor is business logic: let's centralize all the business logic in the process and avoid having it in the BFF

Reply:
I discussed this point with @galales e @beetlecrunch. 
The fact that those values are taken from the previous descriptor is not related to the core logic, but it's a FE-related comodity. It's preferable to be kept this way in order to maintain flexibility